js/containers/report/iframe-standalone-app.js

```diff
@@ -54,7 +54,7 @@ class IframeStandaloneApp extends PureComponent {
       state = answer.get("answer");
     }
     return (
-      <InteractiveIframe src={url} state={state} style={{border: "none"}} width="100%" height="100%" />
+      <InteractiveIframe src={url} state={state} answer={answer} style={{border: "none"}} width="100%" height="100%" />
```

Comment:
This `answer` piece seems unrelated, but I'll take your word for it that it's the right thing to do. 😉

Reply:
This is necessary for handleGetAttachmentsAPI to work. `answer` is similar to `answerMeta` in Activity Player, it contains attachments folder and info.